src/org/opendatakit/validate/StubReference.java

```diff
@@ -62,8 +80,17 @@ public String getURI() {
 
     @Override
     public String getLocalURI() {
+
+        String extension = null;
+        if (mURI != null) {
+            extension = mURI.substring(mURI.length() - 4).toLowerCase();
+        }
         try {
-            return getClass().getClassLoader().getResource("fake-itemset.xml").toURI().getPath();
+            if (extension.equals(".csv") || extension.equals(".xml")) {
```

Comment:
Technically it's the connector (`jr://file` vs. `jr://file-csv` that differentiates between the two. I think this is ok, though.

Reply:
I'd feel better doing it correctly.